mathesar_ui/src/systems/table-view/table-inspector/table/links/LinkSection.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  import { Button, Icon } from '@mathesar/component-library';
+  import {
+    iconLinksFromOtherTables,
+    iconLinksInThisTable,
+  } from '@mathesar/icons';
+  import { MissingExhaustiveConditionError } from '@mathesar/utils/errors';
+  import { modal } from '@mathesar/stores/modal';
+  import LinkTableModal from '@mathesar/systems/table-view/link-table/LinkTableModal.svelte';
+  import type { TableLink, TableLinkType } from './utils';
+  import LinkItem from './LinkItem.svelte';
+
+  export let links: TableLink[];
+  export let type: TableLinkType;
+  export let showCreateLinkButton = false;
+
+  const linkTableModal = modal.spawnModalController();
+
+  $: icon = (() => {
+    switch (type) {
+      case 'in_this_table':
+        return iconLinksInThisTable;
+      case 'from_other_tables':
+        return iconLinksFromOtherTables;
+      default:
+        throw new MissingExhaustiveConditionError(type);
+    }
+  })();
+
+  $: title = (() => {
+    switch (type) {
+      case 'in_this_table':
+        return 'In this table';
+      case 'from_other_tables':
+        return 'From other tables';
+      default:
+        throw new MissingExhaustiveConditionError(type);
+    }
+  })();
+</script>
+
+<div class="link-section-container">
+  <div class="header">
+    <div class="left">
+      <Icon {...icon} />
+      <span>{title}</span>
+    </div>
+    {#if showCreateLinkButton}
+      <div class="right">
+        <Button
+          appearance="plain"
+          class="padding-zero"
+          on:click={() => linkTableModal.open()}
+        >
+          Create Link
+        </Button>
```

Comment:
The figma design has this button in the content with a different styling, and it is not as part of the header.